### DISCUSSION TITLE
Alerts about Google Mini speakers

### DISCUSSION THREAD

**Discussion by adm2k** [netalertx/NetAlertX#735] [Q&A]
On a daily basis and multiple times a day I am getting email alerts that my two Google Home Mini's are "down" - when they are not. I have tried to find a reason for this with no success.  

I suppose I could just remove them from monitoring but would rather not.

Anyone else experiencing this or have any suggestions?

Many thanks!

**Comment by jokob-sk** [comment]
Hi @adm2k ,

I got a Google Home Mini and it disconnects pretty often for me too: 
![image](https://github.com/user-attachments/assets/00aa9e7a-94b4-4052-bc29-27cf65a2fb13)
